include/android-boottime-definition.jinja2

```diff
@@ -0,0 +1,10 @@
+        - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
```

Comment:
in the changelog explain why this deserves its own file. That it will be used by more tests, that will be added.